pkgs/by-name/gl/glycin-loaders/package.nix

```diff
@@ -47,14 +50,23 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   buildInputs = [
+    gobject-introspection
     gtk4 # for GdkTexture
     cairo
+    lcms2
     libheif
     libxml2 # for librsvg crate
+    librsvg
```

Comment:
I wonder if we need to adjust the sandbox path adjustment for fontconfig (likely for rendering fonts in svg).